addons/web/static/src/js/core/domain.js

```diff
@@ -354,6 +354,11 @@ var Domain = collections.Tree.extend({
                 rightBoundaryParams = t ? {days: -1} : {};
                 offsetPeriodParams = {days: -365};
                 return makeInterval();
+            case 'last_5_years':
+                leftBoundaryParams = {years: -5};
+                rightBoundaryParams = _.extend(t ? {days: -1} : {}, {month: 1, day: 1});
```

Comment:
@Polymorphe57 Shouldn't it be like last_year? date should not start with 01-01-year? else domain will be prepared like context_today + relativedelta(day=current day, month=current month)

Reply:
I have asked to FGI and apparently the behavior should be similar to last_7_days,... 
The idea is to have a period of five years exactly (roughly 5 times 365 days accomplished).
It might seem weird but I think there is no other good option.

Reply:
Thank you